src/mame/igs/igs_m027.cpp

```diff
@@ -1446,6 +1446,17 @@ void igs_m027_state::init_chessc2()
 {
 	chessc2_decrypt(machine());
 	m_igs017_igs031->sdwx_gfx_decrypt();
+	m_igs017_igs031->tarzan_decrypt_sprites(0, 0);
+
+	// bypass IGS025 'version' check
+	u16* ROM = (u16*)memregion("user1")->base();
+	ROM[0x207d8/2] ^= 0x0100;
+	// bypass external ROM checksum
+	u16* ROM2 = (u16*)memregion("maincpu")->base();
+	ROM2[(0x16a/2)] ^= 0x1000;
```

Comment:
There’s an object finder for the `user1` region in the class already, and these two regions are both 32 bits wide, so interpreting them as `u16` will cause the code to break on big Endian hosts.

Please access the external ROM through `m_external_rom` and use `->as_u32(...)` to get a reference into the internal ROM.  Something like:
```c++
	// bypass IGS025 'version' check
	m_external_rom[0x207d8/4] ^= 0x00000100;
	// bypass external ROM checksum
	u32 *ROM2 = &memregion("maincpu")->as_u32();
	ROM2[(0x168/4)] ^= 0x10000000;
```

(I haven’t tested that, I’m doing the calculations in my head.)